Modules/TPC/src/Utility.cxx

```diff
@@ -53,6 +55,22 @@ std::vector<TCanvas*> toVector(std::vector<std::unique_ptr<TCanvas>>& input)
   return output;
 }
 
+void fillCanvases(const o2::tpc::CalDet<float>& calDet, std::vector<std::unique_ptr<TCanvas>>& canvases, std::unordered_map<std::string, std::string>& params, const std::string paramName)
+{
+  auto vecPtr = toVector(canvases);
+  o2::tpc::painter::makeSummaryCanvases(calDet,
+                                        std::stoi(params[(paramName + "NBins")]), std::stof(params[(paramName + "XMin")]), std::stof(params[(paramName + "XMax")]),
+                                        true,
+                                        &vecPtr);
+}
+
+void clearCanvases(std::vector<std::unique_ptr<TCanvas>>& canvases)
+{
+  for (const auto& canvas : canvases) {
+    canvas.get()->Clear();
```

Comment:
Is the get really needed?
```suggestion
    canvas->Clear();
```

Reply:
Indeed not. Smart pointers are smarter than I thought. ;)